src/TRestAxionMagneticField.cxx

```diff
@@ -705,13 +705,14 @@ TCanvas* TRestAxionMagneticField::DrawTracks(TVector3 vanishingPoint, Int_t divi
 void TRestAxionMagneticField::LoadMagneticFieldData(MagneticFieldVolume& mVol,
                                                     std::vector<std::vector<Float_t>> data) {
     mVol.field.resize(mVol.mesh.GetNodesX());
-    for (int n = 0; n < mVol.field.size(); n++) {
+    for (unsigned int n = 0; n < mVol.field.size(); n++) {
```

Comment:
All these should be `size_t` in my opinion, not `unsigned int` (to match the type returned by `std::vector::size()` which is an alias for unsigned int but...).

Reply:
So, please, commit the change

Reply:
> So, please, commit the change

It's a general guideline that should apply to all instances where this pattern is repeated (in multiple current PRs atm). I just want to point it out because it takes the same amount of effort on your side to change `int` into `unsigned ìnt` or `size_t`. If this was a quick change I would make it as a suggestion in a comment like I've done in multiple other comments, but in this case doing it for a single line does not accomplish anything, so I didn't. 

I think making a **suggestion** (which does not block the PR either) on a PR should not bind you to actually implementing the suggested changes, in general I think this weight should fall on the PR's author, which can either do it or not at his own prerrogative.

I think expecting reviewers to implement their suggestions will discourage people from making suggestions in general, but this is just my opinion.

Reply:
Ok, if it is just a suggestion then just approve the PR and let the author decide if the suggestion should be implemented or not

Reply:
This PR is about fixing several warnings that improves the quality of the code, and avoids future warnings by implementing a new pipeline that avoids new warnings. If you think the types `size_t` and `unsigned int` should be adjusted, then, please, create a separate PR.